coreneuron/io/lfp.hpp

```diff
@@ -0,0 +1,217 @@
+#ifndef LFP_H
+#define LFP_H
+
+#include <cmath>
+#include <array>
+#include <iostream>
```

Comment:
Including `iostream` in a header is usually not a good sign.
Either it is not used, or the code that needs it should be in a compilation unit, not a header.

Reply:
Done.